lib/interactivity-api.php

```diff
@@ -24,9 +24,31 @@ function gutenberg_reregister_interactivity_script_modules() {
 	wp_register_script_module(
 		'@wordpress/interactivity-router',
 		gutenberg_url( '/build-module/interactivity-router/index.min.js' ),
-		array( '@wordpress/interactivity' ),
+		array(
+			array(
+				'id'     => '@wordpress/a11y',
+				'import' => 'dynamic',
+			),
+			'@wordpress/interactivity',
+		),
 		$default_version
 	);
 }
-
 add_action( 'init', 'gutenberg_reregister_interactivity_script_modules' );
+
+function gutenberg_register_interactivity_script_module_data_hooks() {
+	if ( ! has_filter( 'script_module_data_@wordpress/interactivity-router', array( wp_interactivity(), 'filter_script_module_interactivity_router_data' ) ) ) {
```

Comment:
Can you explain what happens in this line?

Reply:
This is intended to account for Core registering the script module data filters in https://github.com/WordPress/wordpress-develop/pull/7304 to prevent double filtering.

Reply:
I added a comment to the function to clarify what's happening here.